docs/source/changelog.rst

```diff
@@ -1,6 +1,24 @@
 What's new
 ==========
 
+einsteinpy 0.2.1 - 2020-06-15
+-----------------------------
+
+This release will bring a dedicated module for the symbolic calculations, which adds some new computations.
```

Comment:
This minor release would bring improvements and new feature additions to the already existing symbolic calculations module.